community/open-meetings.mdx

```diff
@@ -7,6 +7,38 @@ These are dedicated sync meetings for specific products, as well as open plannin
 
 ## Regular meetings
 
+<MeetingInfo title="Office Hour"
+             schedule="Every Friday effective 16. Feb 2024 until 31. Dec 2024 from 01:05 pm to 02:00 pm CET (UTC +1)"
```

Comment:
Defining CET should be sufficient enough, so I'd remove the "(UTC +1)".

Reply:
changed